ethcore/sync/src/api.rs

```diff
@@ -30,9 +30,8 @@ use std::{
 };
 
 use chain::{
-    sync_packet::SyncPacket::{PrivateTransactionPacket, SignedPrivateTransactionPacket},
     ChainSyncApi, SyncStatus as EthSyncStatus, ETH_PROTOCOL_VERSION_62, ETH_PROTOCOL_VERSION_63,
-    PAR_PROTOCOL_VERSION_1, PAR_PROTOCOL_VERSION_2, PAR_PROTOCOL_VERSION_3,
+    PAR_PROTOCOL_VERSION_1, PAR_PROTOCOL_VERSION_2,
```

Comment:
I'm not sure, that it's correct. What would happen, if someone add another changes, raises the version to v3 again? There would be two conflicting versions of protocol?